HDSAviz/basic_plot.py

```diff
@@ -0,0 +1,122 @@
+"""This is a basic script for plotting an analysis file."""
```

Comment:
Ok i will try updating this to import during our integration task this week.
This is in my TO DO list for now.